Underdwellers Skill/Race expansion.

Drifter7371 opened the pull request:
## About The Pull Request
Gives Underdwellers
- Alchemy 4 
((Taken from Adventurer: Bombadier, 
this should in theory, allow the easy creation of bottlebombs, 
might give them some extra use for all the rocks they get?))
- Masonry 3. 
(Cave people should be able to work stone.)
- Smelting 3. 
(Opens the ability to sell decent quality ingots or use them.)
- Climbing 3 (Raised from 2) 
((Cave people can climb pretty good now.))


## Why It's Good For The Game
Expands the uses of Underdweller and tweaks the skills just a twinge, giving Underdwellers more to work with.
raises the variety of races by two, without giving other races that don't quite make sense access.
(Finally, Cave rats, bats, and mole people.)

Drifter7371 committed:
Update underdweller.dm

Testing a small code change, confidence reasons.

Drifter7371 committed:
MajorTweak

Drifter7371 commented:
> what, no, we're not making a merc class an omni-combat/crafter we just pruned that out

Can you explain the exact issue, besides what can be obtained by simply picking a virtue.
I'm looking for some feedback. It'd help alot.

sneksnek commented:
Mercenaries shouldn't be proficient in crafting skills. Selecting a crafting virtue is not free since it means the character is not using another beneficial virtue. Besides that, expert wrestling was the most mechanically unique thing about underdweller since it was the only merc with it, which is relevant in the current meta of things. Without it, it's fairly underwhelming. One doesn't choose mercenary to go forge.

Drifter7371 commented:
> Mercenaries shouldn't be proficient in crafting skills. Selecting a crafting virtue is not free since it means the character is not using another beneficial virtue. Besides that, expert wrestling was the most mechanically unique thing about underdweller since it was the only merc with it, which is relevant in the current meta of things. Without it, it's fairly underwhelming. One doesn't choose mercenary to go forge.

So the issue is specifically the blacksmithing related talents?
Imagining what to tweak. Could you provide some ideas as to make the class more appealing?

burdwatcher commented:
Please don't give mercs everything they need to be self sufficient, this is +10 skill points in all the things they shouldn't be doing.

AsterTX commented:
So here is some actual feedback because we're actually lacking anything fruitful in the comments. If the essence of them being somewhat self-sufficient is damning to the field then perhaps removing smelting and the smithing since it could be played like a sapper. Even with it removed someone can take the blacksmith apprentice and technically have this all under their belt, not needing the stat changes. 

Bumping wrestling back up could help quite a bit since it was the only class with that high of wrestling. 

But I believe a mining mercenary would be able to keep their gear afloat and not have to run back and forth for repairs to a degree. It makes sense since they ARE people of the caves. There isn't a lot of ways one can actually DO MUCH in those areas.

All in all? It'd probably be some form of sapper which in its self is actually a fun concept.

Drifter7371 commented:
> So here is some actual feedback because we're actually lacking anything fruitful in the comments. If the essence of them being somewhat self-sufficient is damning to the field then perhaps removing smelting and the smithing since it could be played like a sapper. Even with it removed someone can take the blacksmith apprentice and technically have this all under their belt, not needing the stat changes.
> 
> Bumping wrestling back up could help quite a bit since it was the only class with that high of wrestling.
> 
> But I believe a mining mercenary would be able to keep their gear afloat and not have to run back and forth for repairs to a degree. It makes sense since they ARE people of the caves. There isn't a lot of ways one can actually DO MUCH in those areas.
> 
> All in all? It'd probably be some form of sapper which in itself is actually a fun concept. The role itself of being one? You aren't going to be overly chatty either with how much work you'll be doing so... It is a worksman trade with the ability to fend for themselves.

Thanks for the feedback, It's assisted in opening my eyes a bit, I shall adjust the changes.

Drifter7371 committed:
Removes Smithing Skill.

Removes smithing, and adds Bombadier's alchemy stats.
For a sapper-like approach.

sneksnek commented:
> > Mercenaries shouldn't be proficient in crafting skills. Selecting a crafting virtue is not free since it means the character is not using another beneficial virtue. Besides that, expert wrestling was the most mechanically unique thing about underdweller since it was the only merc with it, which is relevant in the current meta of things. Without it, it's fairly underwhelming. One doesn't choose mercenary to go forge.
> 
> So the issue is specifically the blacksmithing related talents? Imagining what to tweak. Could you provide some ideas as to make the class more appealing?

I'm more or less echoing what others have added. I dislike the concept of mercenaries having crafting skills (like the suggested ones) since at the very least, it'll make them self-sufficient, or at the worst, makes dedicated crafting role redundant. The suggested skills are more the former than the latter which is less offensive to the general gameplay but ideally something that should be avoided. Characters needing to seek the crafting roles to receive repairs or items is healthy.

Getting more specific with Underdweller in particular, maintaining their equipment isn't uniquely necessary besides maybe their tool of their trade, the pickaxe. Some skill to repair specifically it would be more acceptable than being able to repair arms and armor and thematically appropriate, but I personally don't find it necessary given the accessibility and efficiency of stone pickaxes.

But I feel like this is beating around the bush. People don't play Underdweller primarily because of the race locking. It may be an unpopular opinion, but I consider Underdweller to be one of if not the best mercenary, mechanically speaking. The reasons why is a whole can of worms that I'll spare your ear.

I believe widening or removing the race lock would do enough to improve popularity. This server doesn't have rigid Underdark lore; it's mostly determined based on vibes and the Forgotten Realms' Underdark. The only other thing I'd suggest is better starting gear.

Apologies in advance if some of what I said is redundant with made changes.

Drifter7371 committed:
fugh

Removes the mention of smithing from the code-notes.
(Not that it effects anything truly.)

Drifter7371 committed:
unimportant

Just correction of something I missed again, nothing to worry about.

Drifter7371 committed:
Fixing potential errors?

IDK errors were going off, trying to figure out why.

Drifter7371 commented:
whoopsies.
